examples/active_support_helper.rb

```diff
@@ -0,0 +1,30 @@
+module ActiveRecord
+  class Base
+  end
+end
+
+module ActiveSupport
+  class TestCase < Minitest::Test
+    def setup(*args, &block)
+    end
+
+    # Remove describe method if present
+    class << self
+      remove_method :describe
+    end if self.respond_to?(:describe) &&
+        self.method(:describe).owner == ActiveSupport::TestCase
+
+    # Add spec DSL
+    extend ::Minitest::Spec::DSL
+
+    if defined?(ActiveRecord::Base)
```

Comment:
Eh, add it to the list. Whenever rspec or minitest add implementation changes, given must update. It's the nature of the library.

I'd recommend this test helper being as minimal as physically possible to keep the test passing.